README.md

```diff
@@ -2,31 +2,304 @@
+Unix, was being created, the developers needed a "starting time". For
+simplicity, they chose the date January 1, 1970, at midnight (or 00:00:00).
+This date time marks the "beginning of time" and is the earliest date a file
+could be created in operating systems and programming languages that use this
+common construct. Meaning, all other dates and times are measured off of this
+datetime. We can also think of the time elapsed from January 1, 1970 at
+00:00:00 as a **Unix epoch time**, which is the number of seconds or
+milliseconds that have passed since the arbitrary start time. An easy resource
+to obtain epoch timestamps is to use the
+[Epoch Converter](https://www.epochconverter.com/)
+which consists of conversion tools to get a specific date as an epoch or vice
+versa.
 
-## Topic 2
+## Java's `Date` Class
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The Java `Date` class represents an instant in time and can be found in the
+`java.util` package. We can construct a `Date` instance in two ways:
+
+```java
+import java.util.Date;
+
+public class DateExample {
+
+    public static void main(String[] args) {
+        // Will instantiate the current date time measured to the nearest millisecond
+        Date date1 = new Date();
+
+        // Will instantiate a date time based on the Unix epoch in milliseconds
+        Date date2 = new Date(1663015601000);
+
+    }
+}
+```
+
+We can also get the time of the `Date` instance by calling the `getTime()`
+method, which will return the number of milliseconds since the arbitrary start
+date of January 1, 1970 at 00:00:00 UTC. (UTC is a timezone that is short for
+"Universal Time Coordinated" and is a primary time standard used to set other
+time zones. It is the same as the "GMT" or "Greenwich Mean Time" timezone.)
+
+```java
+import java.util.Date;
+
+public class DateExample {
+
+    public static void main(String[] args) {
+        // Will instantiate the current date time measured to the nearest millisecond
+        Date date1 = new Date();
+        System.out.println(date1.getTime());    // <-- will print current date time in milliseconds
+
+        // Will instantiate a date time based on the Unix epoch in milliseconds
+        Date date2 = new Date(1663015601000);
+        System.out.println(date2.getTime());    // <-- will print 1663015601000
+
+    }
+}
+```
+
+If we take a look at the Java documentation for the `Date` class here:
+[Java 17 Date Class](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/Date.html),
+we see nearly every constructor and method is deprecated. This is due to the
+shortcomings of the `Date` class. The `Date` class once "allowed the formatting
+and parsing of date strings. Unfortunately, the API for these functions was not
+amenable to internalization." With that said, Java decided to introduce a new
```

Comment:
I think this is supposed to be 'internationalization'?

Reply:
Wow good catch!